web/modules/custom/dbcdk_community_content/src/FieldNormalizer/ImageItemFieldNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Contains ImageItemFieldNormalizer.
+ */
+
+namespace Drupal\dbcdk_community_content\FieldNormalizer;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Normalize the output for an Image field.
+ */
+class ImageItemFieldNormalizer implements FieldNormalizerInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize(FieldItemBase $field) {
+    // Load the file by target_id.
+    /* @var \Drupal\file\Entity\File $image */
+    $image = File::load($field->get('target_id')->getString());
```

Comment:
There difference here is that this will have to access the configuration storage (DB or whatever).

As far as I can tell we have the following options:

- Change this to [`entity_load('File', $target_id)`](https://api.drupal.org/api/drupal/core%21includes%21entity.inc/function/entity_load/8.1.x). That means we can use php-mock to mock the function call
- Implement `ContainerFactoryPluginInterface` and inject an `EntityStorageInterface` for Files